switchmap/poller/snmp/async_snmp_manager.py

```
@@ -0,0 +1,1100 @@
+"""Async SNMP manager class"""
+
+import os
+import sys
```

Ruff (0.12.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)